package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-wifi-reborn",
-  "version": "4.12.1",
```

Comment:
You don't need update the package manually, semantic-release would do automatically